index.html

```diff
@@ -787,10 +787,10 @@ <h3>
           <span><code>app_id</code> member</span>
         </h3>
         <p>
-          The [=MiniApp manifest's=] <code><dfn>app_id</dfn></code> member is a [=string=] that identifies the MiniApp univocally. This member is mainly used for package management, and it supports the update and release process of MiniApp versioning.
+          The [=MiniApp manifest's=] <code><dfn>app_id</dfn></code> member is a case-sensitive [=string=] that identifies the MiniApp uniquely. This member is mainly used for package management, supporting the update and release process of MiniApp versioning.
         </p>
         <p>
-          The value of <code>[=app_id=]</code> is RECOMMENDED <a data-link-type="dfn" href="https://infra.spec.whatwg.org/#string-is">to be</a> a [=string=] defined by the following rule:
+          The value of <code>[=app_id=]</code> SHOULD be defined by the following rule:
```

Comment:
I just commented on the old PR and then saw this. This is generally better, but would again recommend that you make this a MUST or, if not, make a clear statement that the app_id can be any string.